tests/cucumber/features/channels/channel_creation.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Channel creation
+  As an authenticated user
+  I want to create a channel
+
+  Background:
+    Given I am on the home page
+    And I am signed in
+    When I click on the new channels button
+
+  # Test creation and validation of new channels
+  @dev
+  Scenario: Create a valid new channel
+    When I enter "Galaxy" into the title field
+    And I enter "pony+galaxy" into the query field
+    And I submit the new channel form
+    Then channel "title" should display "Galaxy"
+    And channel "query" should display "pony+galaxy"
```

Comment:
Do you think this should also check that the route is correct? As in we actually redirected to a different page.